ck

```diff
@@ -55,7 +55,7 @@ then
 else
   if [[ -n $KUBECONFIG ]]
   then
-    verbose -1 "WARNING: KUBECONFIG is set and is being ignored by $(basename ${0})"
+    verbose -1 "WARNING: KUBECONFIG is set and is being ignored by $CLUSTERPOOL_CONTEXT_NAME"
```

Comment:
If you want to include the name of the context as well, that is OK with me. Eg:
```
    verbose -1  "WARNING: KUBECONFIG is set and is being ignored by $(basename ${0}) (CLUSTERPOOL_CONTEXT_NAME=${CLUSTERPOOL_CONTEXT_NAME})"
```

Reply:
Hi @KevinFCormier I modified the PR as you suggested

Reply:
```
~/9bin % cka list cc
WARNING: KUBECONFIG is set and is being ignored by cka (CLUSTERPOOL_CONTEXT_NAME=applifecycle-ck)
```

Here's the output of the changes on my ENV